src/Avalonia.Base/Media/Fonts/Tables/VerticalHeaderTable.cs

```diff
@@ -0,0 +1,127 @@
+
+        public short MinTopSideBearing { get; }
+
+        public short MinBottomSideBearing { get; }
+
+        public ushort NumberOfVMetrics { get; }
+
+        public short YMaxExtent { get; }
+
+        public static VerticalHeaderTable? Load(IGlyphTypeface fontFace)
+        {
+            if (!fontFace.PlatformTypeface.TryGetTable(Tag, out var table))
+            {
+                return null;
+            }
+
+            var binaryReader = new BigEndianBinaryReader(table.Span);
+
+            return Load(binaryReader);
+        }
+
+        private static VerticalHeaderTable Load(BigEndianBinaryReader reader)
+        {
+            // See OpenType spec for vhea:
+            // | Fixed  | version             | 0x00010000 (1.0)                                                                |
+            // | FWord  | ascender            | Distance from baseline of highest ascender (vertical)                            |
+            // | FWord  | descender           | Distance from baseline of lowest descender (vertical)                            |
+            // | FWord  | lineGap             | typographic line gap (vertical)                                                  |
+            // | uFWord | advanceHeightMax    | must be consistent with vertical metrics                                         |
+            // | FWord  | minTopSideBearing   | must be consistent with vertical metrics                                         |
+            // | FWord  | minBottomSideBearing| must be consistent with vertical metrics                                         |
+            // | FWord  | yMaxExtent          | max(tsb + (yMax-yMin))                                                           |
+            // | int16  | caretSlopeRise      | used to calculate the slope of the caret (rise/run) set to 1 for vertical caret  |
+            // | int16  | caretSlopeRun       | 0 for vertical                                                                   |
+            // | FWord  | caretOffset         | set value to 0 for non-slanted fonts                                             |
+            // | int16  | reserved            | set value to 0                                                                   |
+            // | int16  | reserved            | set value to 0                                                                   |
+            // | int16  | reserved            | set value to 0                                                                   |
+            // | int16  | reserved            | set value to 0                                                                   |
+            // | int16  | metricDataFormat    | 0 for current format                                                             |
+            // | uint16 | numOfLongVerMetrics | number of advance heights in vertical metrics table                              |
+
+            ushort majorVersion = reader.ReadUInt16();
```

Comment:
The version should probably be exposed as a property to match the other tables. Right now those variables are unused.